installer/services.wxs

```diff
@@ -15,8 +15,16 @@
       Impersonate="no"  
       Execute="deferred" 
       FileKey='nssm'
-      ExeCommand='install "Auth0 ADLDAP" "[INSTALLDIR]\node.exe" "\"[INSTALLDIR]\server.js\""'
+      ExeCommand='install "Auth0 ADLDAP" powershell ./start-connector.ps1'
+      Return='asyncWait' />
+
+    <CustomAction
+      Id='SetupConnectorServicePath'
+      Impersonate="no"
+      Execute="deferred"
+      FileKey='nssm'
+      ExeCommand='set "Auth0 ADLDAP" AppDirectory \"[INSTALLDIR]"'
```

Comment:
If this looks weird, it's because escaping quotes is absolute horsepoop in w/e is happening with these files.